src/main/java/nl/ordina/jobcrawler/controller/ResponseCode.java

```diff
@@ -0,0 +1,12 @@
+package nl.ordina.jobcrawler.controller;
+
+import lombok.Getter;
+
+@Getter
+public class ResponseCode {
+    private String ErrorCode;
+
+    public ResponseCode(String errorCode) {
+        ErrorCode = errorCode;
+    }
+}
```

Comment:
From what I gather from the usages, this class is used to create exceptions. Is that true? if yes why not creating the exceptions as their own class?

Reply:
no, it is to create an error message

Reply:
But why not creating custom exceptions? this way it is also readable